packages/time-series-chart/static/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "time series",
+  "icon": "fa fa-bell-o",
+  "parameters": [
+    {
+      "name": "flag",
```

Comment:
I would name this parameter something else, like "Transpose" and make it boolean. When transpose is on then users must provide a column for the series instead using individual columns for the series.